### DISCUSSION TITLE
SEQ Named Instances

### DISCUSSION THREAD

**Discussion by jimpevans** [datalust/seq-tickets#1569] [Help]
We have a need for separate data store for each of our clients. I discovered Named Instances in the docs but then also the warning "Named instances are useful for non-critical environments and low-throughput scenarios. For production deployments, a single instance per server is recommended." We are wondering if there is a resolution or another approach to implement this functionality in a production environment.

**Comment by nblumhardt** [comment]
Hi Jim, thanks for your message. I've replied via the email support thread, as we'll need more information about your goals to help identify a good solution for you.

**Comment by drullo** [comment]
I would also appreciate some clarification on this topic.  Particularly, what does Seq consider to be "low-throughput"?  I have nothing to compare my environment to so I'm not sure if my current traffic load is considered high or low throughput.  Is it based on number of events per time period?  If so, where is that defined?

**Comment by liammclennan** [comment]
Hi @drullo 

To be as fast as possible Seq utilizes all the memory and CPU cycles it can get its digital hands on. If a server has multiple instances of Seq, and they each try to use everything, the result is unpredictable, and hence, not recommended. 

The solution, if you cannot provision a server for production Seq, is to use a virtual server or container.
